### PR TITLE
sd-bus add "sensitive" flag and do not emit properties unnecessarilly

### PR DESCRIPTION
**Pull request by poettering**
Some preparatory work split out of #14096, to merge ahead.

(doesn't do much on its own but add new utility code.)

### CODE REVIEW

File: src/libsystemd/libsystemd.sym [hunk 1]
```diff
@@ -692,4 +692,5 @@ global:
         sd_event_source_get_child_process_own;
         sd_event_source_set_child_process_own;
         sd_event_source_send_child_signal;
+        sd_bus_message_sensitive;
```
Comment (keszybz) [inline src/libsystemd/libsystemd.sym]: Wrong place.

File: src/libsystemd/sd-bus/bus-message.c [hunk 2]
```diff
@@ -45,12 +45,24 @@ static void message_free_part(sd_bus_message *m, struct bus_body_part *part) {
         assert(m);
         assert(part);
 
-        if (part->memfd >= 0)
+        if (part->memfd >= 0) {
+                /* erase this requested, but ony if the memfd is not sealed yet, i.e. is writable */
```
Comment (keszybz) [inline src/libsystemd/sd-bus/bus-message.c]: "this" → "if" ?

File: src/libsystemd/libsystemd.sym (hunk 1 above)
Comment (poettering) [reply src/libsystemd/libsystemd.sym]: hmm?

Comment (poettering) [reply src/libsystemd/libsystemd.sym]: you mean order alphabetically?

Comment (yuwata) [reply src/libsystemd/libsystemd.sym]: ~~I guess this should be in v245.~~

**Comment by poettering** [comment]
force pushed new version, where the .sym file is sorted, and the comment fix, nothing else. ptal

**Pull request merged**